tools/roslaunch/src/roslaunch/nodeprocess.py

```diff
@@ -61,7 +61,7 @@ def _next_counter():
     _counter += 1
     return _counter
 
-def create_master_process(run_id, type_, ros_root, port):
+def create_master_process(run_id, type_, ros_root, port, nworkers, timeout):
```

Comment:
This is public API which can't be changed in breaking way like this. Please make the new arguments optional.